dotCMS/src/main/java/com/dotcms/util/marshal/CustomDotCmsTypeGsonConfigurator.java

```diff
@@ -20,8 +16,9 @@
     @Override
     public void configure(final GsonBuilder gsonBuilder) {
 
-        gsonBuilder.registerTypeAdapter( Payload.class, new PayloadDeserializer() );
+        gsonBuilder.registerTypeAdapter( Payload.class, new PayloadAdapter() );
```

Comment:
If the Deserializer is named, Deserializer. The Serializer should be call Serializer!